### ISSUE TITLE
Abide Required Email & Password Always Invalid

### ISSUE THREAD

**Issue by Knight-Yoshi** [zurb/foundation-sites#7345]
Similar to this issue,  https://github.com/zurb/foundation-sites/issues/7304, required email and password fields are always invalid.

There's also a small syntax error with the Abide Error State example on the site. The is-invalid-input is in the type attribute, not the class
 <input type="text is-invalid-input" required aria-invalid aria-describedby="uuid">

**Comment by johanneslamers** [comment]
Have the same problem. Always invalid on email.

````
<input autocomplete="on" id="fields-emailAddress" name="fields[emailAddress]" placeholder="Email" type="email" value="" required>

**Comment by jamilservicos** [comment]
fix for password input => [#7300](https://github.com/zurb/foundation-sites/pull/7300/files)

example working of email input: `<input type="text" pattern="email" placeholder="Email" id="Email" aria-describedby="registerEmail" required="">`

**Comment by zurbchris** [comment]
Should be fixed with #7590, please comment and submit pr's to the `abide` branch until 6.1 release.
